charts/camunda-platform/templates/connectors/deployment.yaml

```diff
@@ -0,0 +1,93 @@
+{{- if .Values.connectors.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "connectors.fullname" . }}
+  labels: {{- include "connectors.labels" . | nindent 4 }}
+  annotations: {{- toYaml .Values.global.annotations | nindent 4 }}
+spec:
+  replicas: 1
```

Comment:
Let's just customize this since it's easy to do (unless the connectors cannot work with multi replicas by design)

Reply:
Are you sure? It's the same for operate, optimize, and tasklist - explicitly setting it as `1`.

Reply:
I put it in the new revision. But please confirm that we need to parametrize it.

Reply:
as long as the application supports it, we should do it.
there is a GH issue for enhancing this in the other apps already (we want the new components to be better than what we have 😉)